api/agencyservice.yaml

```diff
@@ -198,6 +260,17 @@ components:
               type: boolean
               example: "false"
 
+    AgencyTopicsDTO:
+      type: object
+      properties:
+        id:
+          type: integer
+          format: int64
+          example: 684
+        name:
+          type: string
+          example: "Counselling"
```

Comment:
please add correct example of the topic, counselling is not one of them

Reply:
counseling is the topic id 1 in english on Mariadb, but I will change it to Adoption and fostering a child so it is clearer